TeensyDmx.cpp

```diff
@@ -433,11 +464,24 @@ void TeensyDmx::completeFrame()
             }
             m_newFrame = true;
             break;
+        case State::RDM_RECV:
+        case State::RDM_RECV_CHECKSUM_HI:
+            // Double check the previous partial message was an RDM one
+            if ((m_mode == DMX_IN) &&
+                (reinterpret_cast<uint8_t*>(&m_rdmBuffer)[1] == E120_SC_SUB_MESSAGE)) {
```

Comment:
Why not just do m_rdmBuffer.subStartCode?  It's clearer and doesn't require a dangerous cast.

Reply:
Done. I'd not really thought about it, but was a bit worried if we didn't have a complete packet, but I guess subStartCode just becomes a pointer, and the whole buffer is available if necessary.